modules/ROOT/pages/backup-restore/copy-database.adoc

```diff
@@ -176,11 +176,11 @@ Cannot be combined with `--skip-properties` or `--copy-only-relationship-propert
 
 |--to-format=<format>
 |Set the format for the new database.
-Must be one of `same`, `standard`, `high_limit`, `aligned`.
+Must be one of `same`, `standard`, `high_limit`, `aligned`, block.
```

Comment:
All the others have these ` ticks around them, the added block doesn't

Reply:
Also perhaps do `or` instead of the last comma, as in other places

Reply:
good catch 😄

Reply:
I can't add `or` as it should be as it is in the codebase.